setup.py

```diff
@@ -0,0 +1,50 @@
+"""Setup Stylist for distribution on PyPI."""
+
+import os
+from setuptools import setup, find_packages
+
+# Get the long description from the README file
+with open(
+    os.path.join(os.path.dirname(__file__), 'README.md'),
+    encoding='utf-8',
+) as handle:
+    LONG_DESCRIPTION = handle.read()
+
+setup(
+    name='stylist',
+    # TODO:
+    # For a discussion on single-sourcing the version across setup.py and the
+    # project code, see
+    # https://packaging.python.org/en/latest/single_source_version.html
+    version='0.1',
+    description='Style checking for Fortran, PSyclone DSL, etc',
```

Comment:
I think we are aiming for something rather more wide ranging than just the current usage. The question is do we want the description to cover what is currently implemented or our aspiration?

Reply:
Up to you! You can always change it in future versions.

Reply:
Can we combine the two with something like "Extensible code style checker currently supporting Fortran"?

Reply:
Done.